Zend/zend_alloc.c

```diff
@@ -386,7 +386,7 @@ ZEND_COLD void zend_debug_alloc_output(char *format, ...)
 	va_list args;
 
 	va_start(args, format);
-	vsprintf(output_buf, format, args);
+	vsnprintf(output_buf, sizeof(output_buf), format, args);
```

Comment:
This entire function looks to be unused, if I'm not mistaken. Maybe we can just drop it.